js/modules/rollercoin.js

```diff
@@ -42,23 +42,30 @@ export const blockReward = {
         icon: "rlt.svg"
     },
     "SOL": {
-        dailyReward: 0.05,
-        timePerBlock: 600,
+        dailyReward: 0.1,
+        timePerBlock: 680,
         name: "Solana",
         ticker: "SOL",
         icon: "sol.svg"
     },
     "RST": {
         dailyReward: 150,
-        timePerBlock: 600,
+        timePerBlock: 1890,
         name: "Roller Season Token",
         icon: "rst.svg"
     },
     "TRX": {
-        dailyReward: 90,
-        timePerBlock: 600,
+        dailyReward: 60,
+        timePerBlock: 660,
         name: "Tron",
         ticker: "tron",
         icon: "trx.svg"
     },
+    "LTC": {
+        dailyReward: 0.015,
+        timePerBlock: 1848,
```

Comment:
it's `600` now :)

Reply:
its hard keep value updated ^^

Reply:
LTC TRX are changed time again x.x what should we put here?

Reply:
maybe possible apply an aside button to open a "config window" where is able to insert manually the timeblock minutes ? :O when empty set the "defaults" :O

Reply:
that sounds good to me :) can be a separate pull request
this file should be for defaults